index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+    <label id="name-label">name</label>
+    <input id="name" type="text" placeholder="my name" required="name">
+    <label id="email-label">email</label>
+    <input id="email" type="email" placeholder="enter email" required="email">
+    <label id="number-label">number</label>
+    <input id="number" type="number" placeholder="enter number" min="1" max="100">
+    <select name="confirm" id="dropdown">
+      <option value="yes">Yes</option>
+      <option value="no">No</option>
+    </select>
+    <label>
+      <input name="select-color" value="red" type="radio"/>
+      Red
+    </label>
+    <label>
+      <input name="select-color" value="black" type="radio"/>
+      Black
+    </label>
+
+    <label>
+      <input name="read" value="read" type="checkbox"/>
+      Read
+    </label>
+    <label>
+      <input name="write" value="write" type="checkbox"/>
+      Write
+    </label>
+    <textarea></textarea>
+    <button id="submit" type="submit">submit</button>
+    </form>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
```

Comment:
It's better to use HTML5 semantic tags :nerd_face: (e.g., header, nav), A semantic element clearly describes its meaning to both the browser and the developer by defining the different parts of a web page like (article-aside-details-figcaption-figure-footer-header-main-mark-nav-section-summary-time).
![img_sem_elements](https://user-images.githubusercontent.com/107482707/200424151-441c1d51-3fde-4515-b622-cc63b8f03931.gif)
You can find more information here :point_right:  [https://www.w3schools.com/html/html5_semantic_elements.asp](url)

Reply:
- [ ] Good job ... Your review is awesome :clap: But please make sure to always phrase the comments politely, as suggestions instead of orders. For example: "Please adjust margin to 5px for label element because that would improve the page more accessibility ..." :+1:

Reply:
Kindly use HTML5 semantic tags 🤓 (e.g., header, nav), A semantic element clearly describes its meaning to both the browser and the developer by defining the different parts of a web page like (article-aside-details-figcaption-figure-footer-header-main-mark-nav-section-summary-time).

![img_sem_elements](https://user-images.githubusercontent.com/107482707/200432657-71a41001-fe59-47dc-974b-9830c0dd0822.gif)

You can find more information here 👉 [https://www.w3schools.com/html/html5_semantic_elements.asp](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url)